var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -80,6 +80,8 @@ class Gmsh(CMakePackage):
 
     # https://gmsh.info/doc/texinfo/gmsh.html#Compiling-the-source-code
     # We make changes to the GMSH default, such as external blas.
+    depends_on("libpng", when="+fltk")
+    depends_on("libjpeg", when="+fltk")
```

Comment:
```suggestion
    depends_on("jpeg", when="+fltk")
```

`jpeg` is a virtual with `libjpeg-turbo` is the preferred provider instead of `libjpeg`

Reply:
I will update that thanks !